pkg/experimental/tests/rpm/md5.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
```

Comment:
This seems useful everywhere. Can you move it to pkg/tests instead of experimental?

Reply:
Indeed.  Done.